### ISSUE TITLE
Presto - data type parsing issues

### ISSUE THREAD

**Issue by harshach** [open-metadata/OpenMetadata#2231]
```
2022-01-16 11:45:50,491] INFO     {metadata.ingestion.sink.metadata_rest:208} - Successfully ingested table sf1.dbgen_version
[2022-01-16 11:45:50,492] INFO     {metadata.ingestion.source.sql_source:64} - Table Scanned: local_presto.household_demographics
[2022-01-16 11:45:50,492] ERROR    {metadata.ingestion.source.sql_source:148} - Table Description Error : 
/Users/harsha/Code/test/env/lib/python3.9/site-packages/pyhive/sqlalchemy_presto.py:151: SAWarning: Did not recognize type 'char(15)' of column 'hd_buy_potential'
  util.warn("Did not recognize type '%s' of column '%s'" % (row.Type, row.Column))

/Users/harsha/Code/test/env/lib/python3.9/site-packages/pyhive/sqlalchemy_presto.py:151: SAWarning: Did not recognize type 'char(16)' of column 's_store_id'
  util.warn("Did not recognize type '%s' of column '%s'" % (row.Type, row.Column))
/Users/harsha/Code/test/env/lib/python3.9/site-packages/pyhive/sqlalchemy_presto.py:151: SAWarning: Did not recognize type 'varchar(50)' of column 's_store_name'
  util.warn("Did not recognize type '%s' of column '%s'" % (row.Type, row.Column))
/Users/harsha/Code/test/env/lib/python3.9/site-packages/pyhive/sqlalchemy_presto.py:151: SAWarning: Did not recognize type 'char(20)' of column 's_hours'
  util.warn("Did not recognize type '%s' of column '%s'" % (row.Type, row.Column))
/Users/harsha/Code/test/env/lib/python3.9/site-packages/pyhive/sqlalchemy_presto.py:151: SAWarning: Did not recognize type 'varchar(40)' of column 's_manager'
  util.warn("Did not recognize type '%s' of column '%s'" % (row.Type, row.Column))
/Users/harsha/Code/test/env/lib/python3.9/site-packages/pyhive/sqlalchemy_presto.py:151: SAWarning: Did not recognize type 'varchar(100)' of column 's_geography_class'
  util.warn("Did not recognize type '%s' of column '%s'" % (row.Type, row.Column))
/Users/harsha/Code/test/env/lib/python3.9/site-packages/pyhive/sqlalchemy_presto.py:151: SAWarning: Did not recognize type 'varchar(100)' of column 's_market_desc'
  util.warn("Did not recognize type '%s' of column '%s'" % (row.Type, row.Column))
/Users/harsha/Code/test/env/lib/python3.9/site-packages/pyhive/sqlalchemy_presto.py:151: SAWarning: Did not recognize type 'varchar(40)' of column 's_market_manager'
  util.warn("Did not recognize type '%s' of column '%s'" % (row.Type, row.Column))
/Users/harsha/Code/test/env/lib/python3.9/site-packages/pyhive/sqlalchemy_presto.py:151: SAWarning: Did not recognize type 'varchar(50)' of column 's_division_name'
  util.warn("Did not recognize type '%s' of column '%s'" % (row.Type, row.Column))
/Users/harsha/Code/test/env/lib/python3.9/site-packages/pyhive/sqlalchemy_presto.py:151: SAWarning: Did not recognize type 'varchar(50)' of column 's_company_name'
  util.warn("Did not recognize type '%s' of column '%s'" % (row.Type, row.Column))
/Users/harsha/Code/test/env/lib/python3.9/site-packages/pyhive/sqlalchemy_presto.py:151: SAWarning: Did not recognize type 'varchar(10)' of column 's_street_number'
  util.warn("Did not recognize type '%s' of column '%s'" % (row.Type, row.Column))
/Users/harsha/Code/test/env/lib/python3.9/site-packages/pyhive/sqlalchemy_presto.py:151: SAWarning: Did not recognize type 'varchar(60)' of column 's_street_name'
  util.warn("Did not recognize type '%s' of column '%s'" % (row.Type, row.Column))
/Users/harsha/Code/test/env/lib/python3.9/site-packages/pyhive/sqlalchemy_presto.py:151: SAWarning: Did not recognize type 'char(15)' of column 's_street_type'
  util.warn("Did not recognize type '%s' of column '%s'" % (row.Type, row.Column))
/Users/harsha/Code/test/env/lib/python3.9/site-packages/pyhive/sqlalchemy_presto.py:151: SAWarning: Did not recognize type 'char(10)' of column 's_suite_number'
  util.warn("Did not recognize type '%s' of column '%s'" % (row.Type, row.Column))
/Users/harsha/Code/test/env/lib/python3.9/site-packages/pyhive/sqlalchemy_presto.py:151: SAWarning: Did not recognize type 'varchar(60)' of column 's_city'
  util.warn("Did not recognize type '%s' of column '%s'" % (row.Type, row.Column))
/Users/harsha/Code/test/env/lib/python3.9/site-packages/pyhive/sqlalchemy_presto.py:151: SAWarning: Did not recognize type 'varchar(30)' of column 's_county'
  util.warn("Did not recognize type '%s' of column '%s'" % (row.Type, row.Column))
/Users/harsha/Code/test/env/lib/python3.9/site-packages/pyhive/sqlalchemy_presto.py:151: SAWarning: Did not recognize type 'char(2)' of column 's_state'
  util.warn("Did not recognize type '%s' of column '%s'" % (row.Type, row.Column))
/Users/harsha/Code/test/env/lib/python3.9/site-packages/pyhive/sqlalchemy_presto.py:151: SAWarning: Did not recognize type 'char(10)' of column 's_zip'
  util.warn("Did not recognize type '%s' of column '%s'" % (row.Type, row.Column))
/Users/harsha/Code/test/env/lib/python3.9/site-packages/pyhive/sqlalchemy_presto.py:151: SAWarning: Did not recognize type 'varchar(20)' of column 's_country'
  util.warn("Did not recognize type '%s' of column '%s'" % (row.Type, row.Column))
/Users/harsha/Code/test/env/lib/python3.9/site-packages/pyhive/sqlalchemy_presto.py:151: SAWarning: Did not recognize type 'decimal(5,2)' of column 's_gmt_offset'
  util.warn("Did not recognize type '%s' of column '%s'" % (row.Type, row.Column))
/Users/harsha/Code/test/env/lib/python3.9/site-packages/pyhive/sqlalchemy_presto.py:151: SAWarning: Did not recognize type 'decimal(5,2)' of column 's_tax_precentage'
  util.warn("Did not recognize type '%s' of column '%s'" % (row.Type, row.Column))
```

**Comment by harshach** [comment]
@ayush-shah this seems mostly coming from presto connector but we could use reflection to extract the raw types and use our parser to process the types

**Comment by ayush-shah** [comment]
Fixed with #2270
